packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -896,6 +896,8 @@ abstract class BoxScrollView extends ScrollView {
 /// A [ListView] is basically a [CustomScrollView] with a single [SliverList] in
 /// its [CustomScrollView.slivers] property.
 ///
+/// {@youtube 560 315 https://www.youtube.com/watch?v=LUqDNnv_dh0}
```

Comment:
This video still feels a little bit out of place under the "Transitioning to [CustomScrollView]" heading. Maybe it would be better to move it (with a brief introduction) to the docs of "shrinkWrap"?

Reply:
I agree, it would be great to add it to the already existing shrinkwrap template, here:
https://github.com/flutter/flutter/blob/5012c99df5a2df8f6696992a7d9610c3b84a0738/packages/flutter/lib/src/widgets/scroll_view.dart#L241